web/app/themes/mitlib-parent/header.php

```diff
@@ -0,0 +1,25 @@
+<?php
+/**
+ * Header template for all pages.
+ *
+ * This constructs both the <head> element, as well as everything in the body tag through the close of the <header>
+ * tags.
+ *
+ * @package MITlib_Parent
+ * @since 0.0.1
+ */
+
+namespace Mitlib\Parent;
+
+?><!DOCTYPE html>
+<html <?php language_attributes(); ?>>
+<head>
+    <meta charset="<?php bloginfo('charset'); ?>" />
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <meta name="description" content="The libraries of the Massachusetts Institute of Technology - Search, Visit, Research, Explore" />
+    <link rel="icon" type="image/png" href="<?php echo esc_attr(get_template_directory_uri()); ?>/favicon.ico">
+    <title><?php wp_title('|', true, 'right'); ?></title>
+    <?php wp_head(); ?>
+</head>
+<body <?php body_class(); ?>>
+    <div class="wrap-page">
```

Comment:
I wonder if we can rework these files to not separate the <body> and wrap page <div>? If I understand how this works, maybe we could move them to page-home and page files (and also move the final closing tags out of footer)?

If that is messier than I am suggesting, we can set this aside for now. The reason I am suggesting this is to prevent having to look in multiple files to mentally understand how the html is being constructed. Looking at this without having to look across additional files I don' know if this is going to result in invalid HTML or if some other file handles closing the tags.

Reply:
We can certainly change the dividing line between what's in header/footer and what's in the page template.

My suspicion, though, is that this boundary is going to be hard to place in a way that avoids the issue entirely. The current [header.php](https://github.com/MITLibraries/MITlibraries-parent/blob/main/header.php) includes even more markup than this stub does, as does [footer.php](https://github.com/MITLibraries/MITlibraries-parent/blob/main/footer.php).

The discussion of [partials and miscellaneous files](https://developer.wordpress.org/themes/template-files-section/partial-and-miscellaneous-template-files/) from the theme developers guide includes examples that generally follow the pattern we've used - although those examples are now several years old (the twenty fifteen theme). A brief glimpse at the [header for twentytwentyone](https://themes.svn.wordpress.org/twentytwentyone/1.6/header.php) seems to be similar, albeit much simpler than what we're dealing with.

I'm happy to look at some alternative division lines - is that something you'd like to be part of this PR, in the Jira ticket, or elsewhere?

Reply:
I'm torn. I think we can make it cleaner and this is one opportunity to do that before things get messy... but also until things get messy it's hard for me to fully understand the problem as I haven't worked closely with these files before.

I think I'm leaning towards this not being in scope for this PR and if either of us have an alternate idea later we PR it for discussion.

Reply:
That sounds good to me - this part of the theme gets built out pretty quickly (maybe in the next PR after this), so it should be possible to have a more meaningful discussion of options at that point.